packages/passport-interface/src/index.ts

```diff
@@ -1,4 +1,18 @@
 export * from "../test/MockFeedApi";
+export * from "./api/requestCheckIn";
+export * from "./api/requestCheckTicket";
+export * from "./api/requestConfirmationEmail";
+export * from "./api/requestCreateNewUser";
+export * from "./api/requestDeviceLogin";
+export * from "./api/requestDownloadAndDecryptStorage";
+export * from "./api/requestEncryptedStorage";
+export * from "./api/requestIssuedPCDs";
+export * from "./api/requestLogToServer";
+export * from "./api/requestPasswordSalt";
+export * from "./api/requestPendingPCD";
+export * from "./api/requestUploadEncryptedStorage";
+export * from "./api/requestUser";
+export * from "./api/requestVerifyToken";
```

Comment:
each implementation of a function that calls the pcdpass api has its own file now